Automattic/wp-calypso
# Plugins: Use typescript plugin selectors in plugins

## Proposed Changes

This PR updates the main Plugins code to use the new TypeScript plugins selectors which were developed beginning in https://github.com/Automattic/wp-calypso/pull/72363.

## Testing Instructions



Check that the following works:
Plugin details page (/plugins/:plugin_slug)
* Activating/deactivating
* Enabling/disabling autoupdate
* Updating

Site details page (/plugins/manage/:site_slug)
* Activating/deactivating via toggle
* Enabling/disabling autoupdate via toggle
* Updating all plugins via primary update button
* Updating one plugin via update link
* Activating/deactivating via "edit all"
* Enabling/disabling autoupdate via "edit all"
* Updating via "edit all"
* Removing via "edit all"
* Removing via ellipsis

Plugins manage page
Site details page (/plugins/manag)
* Activating/deactivating via toggle
* Enabling/disabling autoupdate via toggle
* Updating all plugins via primary update button
* Updating one plugin via update link
* Activating/deactivating via "edit all"
* Enabling/disabling autoupdate via "edit all"
* Updating via "edit all"
* Removing via "edit all"
* Removing via ellipsis


## Pre-merge Checklist
